lib/metaractor/spec.rb

```diff
@@ -17,7 +17,10 @@ def context_creator(error_message: nil, error_messages: [], errors: [], valid: n
           result.invalidate!
         end
 
-        if !result.errors.empty? || (success != nil && !success) || (failure != nil && failure)
+        if !result.errors.empty? ||
+            result.invalid? ||
+            (success != nil && !success) ||
+            (failure != nil && failure)
```

Comment:
It's weird to me that these are two different things in the interactor and one is not alias to the other.

Reply:
They basically function as aliases unless you do something crazy and specify them both.